### DISCUSSION TITLE
Craft Cloud and Control Panel branding

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cloud#40] [Q&A] [answered]
A project being migrated is using the /storage/rebrand path to load a custom icon and logo for the Control Panel as it is a Craft CMS Pro site, both assets do not appear to be loading on Craft Cloud (it is working locally).

Is there anything special that needs to be done to have this load on Craft Cloud?

**Comment by timkelty** [comment, marked as answer]
@jamesmacwhite as rebrand assets aren't traditional Craft assets, you'll need to tell Craft Cloud where you have them using `CRAFT_REBRAND_PATH`. See https://github.com/craftcms/cloud/discussions/10#discussioncomment-9876939 for details.

We have documented this in a new [Rebrand Assets on Craft Cloud](https://craftcms.com/knowledge-base/cloud-rebrand-assets) article!

**Comment by jamesmacwhite** [comment]
Ah thanks for this, good to know. I'll add that env var to resolve the issue.
